.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+#### Link this PR to an issue
+Fixes #ISSUE-NUMBER (eg. Fixes #147)
+
+#### Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+#### Describe the changes you have made in this PR -
+
+#### Screenshots of the changes (If any) -
+
+#### How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+
+#### Checklist:
+
+- [ ] My code follows the style guidelines of this project and performed a self-review of my own code
```

Comment:
> the style guidelines of this project

Can this be linked to the guidelines?

Reply:
do you mean by providing a link to guidelines page from there?

Reply:
Yeah, if that's possible

Reply:
yes, it's markdown so it's possible!  

But when a user creates PR, it is basically in write mode and it will look like this to the user. 
![image](https://user-images.githubusercontent.com/55848322/155949727-274da076-6cad-4674-9907-0c467324ac60.png)

In any way to look at the guidelines, the user has to copy-paste the link, or go in preview mode and click on the link.

Reply:
I think that's ok. Just to make it visible also for other people looking at these PRs.

Reply:
yeah! can you share me link to the guidelines.

Reply:
:D I don't think they exists (yet). Maybe leave this out for now. We could come up with a draft in a new issue maybe.

Reply:
okay cool!